File: week6/Baekjoon28290/dohyungLee/baekjoon_28290.py
def main() :
    strInput = input()
    classifyInput(strInput)

# 입력에 따라 정렬해주는 함수
def classifyInput(strInput) :
    
    # 받은 입력에 따라 분류하여 출력
    if strInput == "fdsajkl;" or strInput == "jkl;fdsa" :
        print("in-out")
    elif strInput == "asdf;lkj" or strInput == ";lkjasdf" :
        print("out-in")
    elif strInput == "asdfjkl;" :
        print("stairs")
    elif strInput == ";lkjfdsa" :
        print("reverse")
    else :
        print("molu")

if __name__ == "__main__" :
    main()

File: week6/Baekjoon28290/seungjunLee/Baekjoon28290.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');

let typing = input[0];
let type = ['fdsajkl;', 'jkl;fdsa', 'asdf;lkj', ';lkjasdf', 'asdfjkl;', ';lkjfdsa'];
let typeName = ['in-out', 'out-in', 'stairs', 'reverse', 'molu'];

switch(typing){
    case type[0] : 
    case type[1] : console.log(typeName[0]); break;
    case type[2] : 
    case type[3] : console.log(typeName[1]); break;
    case type[4] : console.log(typeName[2]); break;
    case type[5] : console.log(typeName[3]); break;
    default : console.log(typeName[4]); break;
}